### PR TITLE
Add Webhook.ApiVersion.

### PR DESCRIPTION
**Pull request by ascherkus**

### CODE REVIEW

**Comment by ascherkus** [comment]
Actually hold off -- I think this should probably be a `*string` and not a `string` (Shopify doesn't like empty string in POST requests). If you'd rather just not want this PR that's fine too :)

File: webhook.go
Comment (oliver006) [inline webhook.go:38]: Can you change this to `ApiVersion` please?

**Comment by oliver006** [comment]
Thanks for your PR!
I recently joined as a maintainer and so things should hopefully move a bit quicker soon.

File: webhook.go
Comment (ascherkus) [reply webhook.go]: Done

**Review by oliver006 — APPROVED** [review]
🥳

**Pull request merged**
